docs/advanced/custom-templates.md

```diff
@@ -159,7 +156,7 @@ You can always save your `template_render` function in another file and import i
 - `dara.core.visual.components.sidebar_frame.SidebarFrame`
 - `dara.core.visual.components.topbar_frame.TopbarFrame`
 
-These are used internally to construct the `default`, `top` and `top-menu` templates. However, you can use them to construct your own custom templates. As an example, you might want to customise the look of your app by using a different logo. To achieve that, put a logo image in the `static` folder alongside a `favicon.ico` file.
+These are used internally to construct the `default`, `top` and `top-menu` templates. However, you can use them to construct your own custom templates. As an example, you might want to customize the look of your app by using a different logo. To achieve that, put a logo image in the `static` folder alongside a `favicon.ico` file.
```

Comment:
@Roman-Kornev do we need to change the docs for the path to the logo?

Reply:
No, that issue was separate. It should now correctly propagate without user input.